components/Leaf/HearingsLeaf.js

```diff
@@ -0,0 +1,17 @@
+import { Image } from "react-bootstrap"
```

Comment:
nit: we can rename this to `Leaf`

Reply:
**Image** is the name of the React-Bootstrap component itself. The component created for the project is called Leaf (which uses the Image component). I could import Image as Leaf in the Leaf component. Also, HearingsLeaf.js no longer exists. I combined it with the existing Leaf component. Did I misunderstand you?